distro/run/assembly/resources/run.bat

```diff
@@ -0,0 +1,157 @@
+@echo off
+
+SET BASEDIR=%~dp0
+SET PARENTDIR=%BASEDIR%..\
+SET DEPLOYMENTDIR=%PARENTDIR%configuration/resources
+SET APPNAME=Camunda Run
+
+IF [%~1]==[start] GOTO Startup
+IF [%~1]==[stop] GOTO Stop
+IF [%~1]==[help] GOTO Help
+IF [%~1]==[] GOTO Help
+
+:Startup
+REM remove argument
+SHIFT
+
+REM setup the JVM
+IF "x%JAVA_HOME%" == "x" (
+  SET JAVA=java
+  ECHO JAVA_HOME is not set. Unexpected results may occur.
+  ECHO Set JAVA_HOME to the directory of your local JDK to avoid this message.
+) ELSE (
+  IF NOT EXIST "%JAVA_HOME%" (
+    ECHO JAVA_HOME "%JAVA_HOME%" path doesn't exist
+    GOTO :EOF
+  ) ELSE (
+    IF NOT EXIST "%JAVA_HOME%\bin\java.exe" (
+      ECHO "%JAVA_HOME%\bin\java.exe" does not exist
+      GOTO :EOF
+    )
+    ECHO Setting JAVA property to "%JAVA_HOME%\bin\java"
+    SET JAVA="%JAVA_HOME%\bin\java"
+  )
+)
+
+IF NOT "x%JAVA_OPTS%" == "x" (
+  ECHO JAVA_OPTS: %JAVA_OPTS%
+)
+
+REM set environment parameters
+SET webappsPath=%BASEDIR%webapps
+SET restPath=%BASEDIR%rest
+SET swaggerPath=%BASEDIR%swaggerui
+SET examplePath=%BASEDIR%example
+SET classPath=%PARENTDIR%configuration\userlib,%PARENTDIR%configuration\keystore
+SET optionalComponentChosen=false
+SET restChosen=false
+SET swaggeruiChosen=false
+SET productionChosen=false
+SET detachProcess=false
+SET configuration=%PARENTDIR%configuration\default.yml
+
+
+REM inspect arguments
+:Loop
+IF [%~1]==[] GOTO Continue
+
+IF [%~1]==[--webapps] (
+  SET optionalComponentChosen=true
+  SET classPath=%webappsPath%,%classPath%
+  ECHO WebApps enabled
+)
+
+IF [%~1]==[--rest] (
+  SET optionalComponentChosen=true
+  SET restChosen=true
+  SET classPath=%restPath%,%classPath%
+  ECHO REST API enabled
+)
+
+IF [%~1]==[--swaggerui] (
+  SET optionalComponentChosen=true
+  SET swaggeruiChosen=true
+  SET classPath=%swaggerPath%,%classPath%
+  ECHO Swagger UI enabled
+)
+
+IF [%~1]==[--example] (
+  SET optionalComponentChosen=true
+  SET classPath=%examplePath%,%classPath%
+  ECHO Invoice Example included - needs to be enabled in application configuration as well
+)
+
+IF [%~1]==[--production] (
+  SET productionChosen=true
+  SET configuration=%PARENTDIR%configuration\production.yml
+)
+
+IF [%~1]==[--detached] (
+  SET detachProcess=true
+)
+
```

Comment:
```suggestion

IF [%~1]==[--help] (
  GOTO Help
)
```

We are missing the help parameter. The `:Help` section is not reachable.

Reply:
I think you are getting to the Help section by calling `run.bat help`

Reply:
Is there a reason why we are hiding this argument for users of the `start` scripts? Not sure if it is of much use in an internal script.

Reply:
https://github.com/camunda/camunda-docs-manual/pull/1151#discussion_r807802209

Reply:
Let's keep the `help` discussion here since it's more related to the implementation. I agree that exposing a `--help` argument in the `start.*` scripts is desirable. The annoying thing is, it requires a lot more logic in the SHELL scripts than in the BAT scripts :smile: . But I'll implement it.

Reply:
@tmetzke regarding https://github.com/camunda/camunda-docs-manual/pull/1151#discussion_r807813814 I think `--help` should trump any other arguments. That's the usual behavior in scripts. And it makes sense since it's better to clarify to a user what the script does, in case they passed some arguments that they don't understand.

Reply:
I added the `--help` argument/